src/shared/components/DashboardRoute.tsx

```diff
@@ -59,30 +58,35 @@ class DashboardRoute extends PureComponent<Props> {
     const dashboardID = this.props.match.params.dashboardID
     dashboardVisit(dashboardID, new Date().getTime())
     event('Dashboard Visit', {dashboardID})
-    const urlVars = qs.parse(this.props.location.search, {
-      ignoreQueryPrefix: true,
-    })
 
     // always keep the dashboard in sync
     if (dashboard !== dashboardID) {
       updateDashboard(dashboardID)
     }
 
+    const params = new URLSearchParams(window.location.search)
+    const vars = params
+      .entries()
+      .filter(([k, _v]) => /(^vars\[)(.+)(\]$)/g.test(k))
```

Comment:
can this be a const somewhere since i can't read regex to save my life?

Reply:
it's not going to be any more readable, i can guarantee that